src/logic/asset_graph.py

```
@@ -1,72 +1,105 @@
+"""Asset relationship graph service backed by the relational database."""
+
+from __future__ import annotations
+
 from typing import Any, Dict, List, Optional, Tuple
```

Ruff (0.14.3)

`typing.Dict` is deprecated, use `dict` instead

(UP035)